pkg/apis/keycloak/v1alpha1/keycloakrealm_types.go

```diff
@@ -7,6 +7,10 @@ import (
 // KeycloakRealmSpec defines the desired state of KeycloakRealm.
 // +k8s:openapi-gen=true
 type KeycloakRealmSpec struct {
+	// When set to true, this KeycloakRealm will be marked as unmanaged and not be managed by this operator.
+	// It can then be used for targeting purposes.
+	// +optional
+	Unmanaged bool `json:"unmanaged,omitempty"`
```

Comment:
If you agree with my earlier comment about removing `unmanaged` field, we could rename this to `external.enabled` for symmetry reasons with `Keycloak` CR.

Reply:
Please disregard this, see https://github.com/keycloak/keycloak-operator/pull/247#discussion_r508239294